### DISCUSSION TITLE
Removed by op

### DISCUSSION THREAD

**Discussion by JoeyBe1** [community/community#124373] [Accessibility]
Removed by OP

**Comment by samus-aran** [comment]
Hi @JoeyBe1 👋 

We can confirm that you have been successful with your application and your academic benefits have been granted 🎉 
Start enjoying GitHub Education 🌟:

- Connect with a local Campus Expert. For more information on campus experts, see "[About Campus Experts](https://education.github.com/students/experts)."
- Explore and claim offers for free industry tools from the [Student Developer Pack](https://education.github.com/pack).
- See upcoming in-person and virtual events for students, curated by GitHub Education and student leaders.
- View assignments from [GitHub Classroom](https://classroom.github.com/) with upcoming due dates.
- Stay in the know with [Campus TV](https://www.twitch.tv/githubeducation) episodes. Campus TV is created by GitHub and student community leaders and can be watched live or on demand.
- Discover and contribute to student-created repositories from GitHub Community Exchange. For more information, see "[About GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange)."


Lastly thank you for your feedback, we've shared this experience with our internal team. 🙂

**Reply by katannshaw** [reply]
@samus-aran I'm having the exact same issue but I'm not seeing that this issue was solved in this post. I'm a 100% online student, so my student ID is online. There's no way for me to take a good photo of the document, and there's no way for me to upload a document. I'm not a blind user, however I'm finding this form to be non-accessible and very frustrating.

Are you planning on adding a way for students to upload documents for their student id in the near future? Thanks for any help.

Kat Shaw

**Comment by cehfisher** [comment]
Hi @katannshaw - I'm going to close out this discussion, as we have resolved the original author's issue and are working directly with you to resolve your issue. We also have filed internal issues to track changes to this process, plus create documentation on how to support others in your situation. If you or anyone else has follow-up questions, please reach out!

**Comment by katannshaw** [comment]
Hi @cehfisher That sounds perfect. Thanks for you and Sydney's help!

Kat
